docs/faq.rst

```diff
@@ -1156,7 +1156,7 @@ full when Borg starts a write command like `borg create`, it will abort
 immediately and the repository will stay as-is.
 
 If you run a backup that stops due to a disk running full, Borg will roll back,
-delete the new new segment file and thus freeing disk space automatically. There
+delete the new segment file and thus freeing disk space automatically. There
```

Comment:
I can't tell if this doubling is intentional or accidental

Reply:
accidental